src/components/simpleInterest/simpleInterest.js

```diff
@@ -1,5 +1,57 @@
 import "./simpleInterest.css";
+import { useState } from "react";
 const SIcomponent = () => {
-  return (<div>this is SI wala page </div>);
+  const [principle, setPrinciple] = useState(0);
+  const [rate, setRate] = useState(0);
+  const [time, setTime] = useState(0);
+  const [si, setSi] = useState(0);
+  const [amount, setAmount] = useState(0);
```

Comment:
instead of 5 useStates, 
You can use single ```useState``` with the initial state as ```{time:0,si:0,.....}```

Reply:
Wow thanks will do that